csharp/src/Microsoft.Azure.EventHubs.Processor/PartitionContext.cs

```diff
@@ -22,6 +22,28 @@ internal PartitionContext(EventProcessorHost host, string partitionId, string ev
             this.ThisLock = new object();
         }
 
+        /// <summary>
+        /// Returns the current offset of the partition.
+        /// </summary>
+        public string Offset
```

Comment:
>public string Offset [](start = 8, length = 20)

just dog public string Offset { get; private set;}
Also if we are exposing this, then we should agree on that null is ok for Offset (currently I don't think the class initialize it)